gluecodium/src/main/resources/templates/dart/DartProperty.mustache

```diff
@@ -28,7 +28,8 @@
 }}{{>dart/DartFunctionBody}}
 {{/getter}}
 {{#setter}}
-{{#unless skipDocs}}{{>dart/DartDocumentation}}
+{{#unless skipDocs}}{{>dart/DartFunctionDocs}}
```

Comment:
Please use `{{>dart/DartFunctionDocs}}` in line 23 above as well. It does not affect the output, but it's needed for template consistency.

Reply:
If I use `DartFunctionDocs` template in `getter` code generation (line 23), It will generate duplicate documentation in getter and setter of dart language only. 
It will be inconsistent with current outputs of C++/Java/etc.